server/synthesizer.py

```diff
@@ -91,11 +92,26 @@ def load_wavernn(self, lib_path, model_path, model_file, model_config, use_cuda)
         ).cuda()
 
         check = torch.load(model_file)
-        self.wavernn.load_state_dict(check['model'])
+        self.wavernn.load_state_dict(check['model'], map_location="cpu")
         if use_cuda:
             self.wavernn.cuda()
         self.wavernn.eval()
 
+    def load_pwgan(self, lib_path, model_file, model_config, use_cuda):
+        sys.path.append(lib_path) # set this if TTS is not installed globally
```

Comment:
This comment is talking about TTS but it's actually about the PWGAN repo right?

Reply:
I guess it was copied from the WaveRNN one, but then that should be fixed too? :P

Reply:
yep :) copy-paste gods will know.